flask_appbuilder/models/sqla/interface.py

```diff
@@ -158,61 +162,61 @@ def is_file(self, col_name):
 
     def is_string(self, col_name):
         try:
-            return isinstance(self.list_columns[col_name].type, sa.types.String)
+            return _is_sa_type(self.list_columns[col_name].type, sa.types.String)
```

Comment:
I'm confused at the code *before* this change, why to we expect `isinstance` to fail? If it's really a thing, maybe `_is_sa_type` should handle it though.

Reply:
I believe these type checks are used as process of elimination to map model fields to WTForm fields.  Each model field iterates through all the types checks until it finds the one that returns True, and then do the conversion accordingly ([ref here](https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/forms.py#L37-L50)).

So if the type didn't exist, then the field doesn't get processed and that breaks the view page.

Reply:
It's a fail safe for self.list_columns[col_name].type